manifests/r/RubyInstallerTeam/RubyWithDevKit/3/2/3.2.0-1/RubyInstallerTeam.RubyWithDevKit.3.2.installer.yaml

```diff
@@ -0,0 +1,37 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: RubyInstallerTeam.RubyWithDevKit.3.2
+PackageVersion: 3.2.0-1
+MinimumOSVersion: 10.0.0.0
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /verysilent /allusers
+  SilentWithProgress: /silent /allusers
```

Comment:
```suggestion
```
We don't need silent switches when Installer type is set to `inno`. We'll add scope specific flags at the installer level.